charts/argo-workflows/values.yaml

```diff
@@ -170,15 +170,6 @@ controller:
   workflowNamespaces:
     - default
 
-  # -- Specifies the container runtime interface to use (one of: `docker`, `kubelet`, `k8sapi`, `pns`, `emissary`)
-  ## Ref: https://argoproj.github.io/argo-workflows/workflow-executors/
-  containerRuntimeExecutor: emissary
-  # -- Specifies the executor to use. This has precedence over `controller.containerRuntimeExecutor`.
-  containerRuntimeExecutors: []
-    # - name: emissary
-    #   selector:
-    #     matchLabels:
-    #       workflows.argoproj.io/container-runtime-executor: emissary
```

Comment:
`containerRuntimeExecutor` and `containerRuntimeExecutors` are no more supported. :eyes:
ref: https://argoproj.github.io/argo-workflows/upgrading/#upgrading-to-v34
<img width="733" alt="Screen Shot 2022-09-19 at 21 04 31" src="https://user-images.githubusercontent.com/30188755/191013208-fa7c812f-c53c-488a-aff6-b113e6696cc0.png">

ref: https://github.com/argoproj/argo-helm/actions/runs/3082197312/jobs/4981596780#step:13:265
<img width="1090" alt="Screen Shot 2022-09-19 at 21 33 57" src="https://user-images.githubusercontent.com/30188755/191018657-5648e003-17ba-44e3-95bc-9afdf498b5cc.png">